src/software/ai/hl/stp/tactic/receiver/receiver_tactic_test.cpp

```diff
@@ -25,86 +25,88 @@ class ReceiverTacticTest
-                               Angle::fromDegrees(5), yield);
-
-            // NOTE: we don't check robotAtPosition for receive and dribble
-            // because the robot is free to adjust itself to best receive
-            // the pass (and dribble). We only care if the robot received the ball.
-            robotReceivedBall(world_ptr, yield);
-
-            while (!tactic->done())
-            {
-                yield("Receiver tactic done but did not receive pass");
-            }
-        }};
-
-    std::vector<ValidationFunction> non_terminating_validation_functions = {};
-
-    runTest(field_type, ball_state, friendly_robots, {}, terminating_validation_functions,
-            non_terminating_validation_functions, Duration::fromSeconds(10));
-}
-
-INSTANTIATE_TEST_CASE_P(
-    PassEnvironmentReceiveAndDribble, ReceiverTacticTest,
-    ::testing::Values(
-        // Robot already at receive point
-        std::make_tuple(Pass(Point(0.0, 0.5), Point(2, 2), 4),
-                        RobotStateWithId{
-                            1, RobotState(Point(2, 2), Vector(0, 0),
-                                          Angle::fromDegrees(0), Angle::fromDegrees(0))}),
-
-        // Robot slighty off from receive point: test 1
-        std::make_tuple(Pass(Point(0.0, 0.5), Point(2, 2), 4),
-                        RobotStateWithId{
-                            1, RobotState(Point(2, 1.5), Vector(0, 0),
-                                          Angle::fromDegrees(0), Angle::fromDegrees(0))}),
-
-        // Robot slighty off from receive point: test 2
-        std::make_tuple(Pass(Point(0.0, 0.5), Point(2, 2), 4),
-                        RobotStateWithId{
-                            1, RobotState(Point(2.5, 2.0), Vector(0, 0),
-                                          Angle::fromDegrees(0), Angle::fromDegrees(0))}),
-
-        // Robot facing away from pass
-        std::make_tuple(Pass(Point(0.0, 0.0), Point(-3, 0), 4),
-                        RobotStateWithId{1, RobotState(Point(-3, 0), Vector(0, 0),
-                                                       Angle::fromDegrees(180),
-                                                       Angle::fromDegrees(0))}),
-
-        // Robot facing towards from pass
-        std::make_tuple(Pass(Point(0.0, 0.0), Point(-3, 0), 4),
-                        RobotStateWithId{
-                            1, RobotState(Point(-3, 0), Vector(0, 0),
-                                          Angle::fromDegrees(0), Angle::fromDegrees(0))}),
-
-        // Robot facing towards pass speedy
-        std::make_tuple(Pass(Point(0.0, 0.0), Point(-3, 0), 5),
-                        RobotStateWithId{1, RobotState(Point(-3, 0), Vector(0, 0),
-                                                       Angle::fromDegrees(0),
-                                                       Angle::fromDegrees(0))})));
+//TEST_P(ReceiverTacticTest, perfect_pass_receiver_test)
```

Comment:
why is this test commented out? Could you tag an issue and disable the test

Reply:
yep was playing with CI because tests were passing locally but not on CI, uncommented